src/caretogether-pwa/src/App.tsx

```diff
@@ -187,7 +196,7 @@ function App() {
     <div className={classes.root}>
       <CssBaseline />
       <Router>
-        <AppBar position="absolute" className={clsx(classes.appBar, open && classes.appBarShift)}>
+        <AppBar position="absolute" className={clsx(classes.appBar, (open && !isMobile) && classes.appBarShift)}>
           <Toolbar className={classes.toolbar}>
```

Comment:
Just a note that there's a merge conflict on this line, make sure you merge `main` into your branch to resolve the conflict.

Reply:
I'll be sure to merge, sort out the conflicts then make this a regular Pull Request